src/driver/dune_style.ml

```diff
@@ -53,6 +53,7 @@ let of_dune_build dir =
                       version = "1.0";
                       libraries = [ lib ];
                       mlds = [];
+                      assets = [] (* TODO *);
```

Comment:
What could we do ?

Reply:
When dune has a notion of asset, we will be able to do something. Right now, nothing. I'll update the comment.

(However, I think something could be done for mld files. I'm not sure we need to spend time on that though)